Web/Admin.UI/src/app/components/account/account-config-form/account-config-form.component.ts

```
@@ -0,0 +1,182 @@
+            else {
+              this.submittedConfiguration.emit({success: false, message: err.message});
+            }
+          }
+        });
+      } else if (this.formMode == FormMode.Edit) {
+          this.accountService.updateUser(user).subscribe({
+            next: (response) => {
+              this.submittedConfiguration.emit({success: true, message: ""});
+            },
+            error: (err) => {
+              if (err.status = 409) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)